tests/test_snapshots.py

```diff
@@ -12,6 +12,15 @@ def test_simple_string(snapshot):
     assert "Loreeeeeem ipsum." == snapshot
 
 
+def test_multiline(snapshot):
+    assert "multi\nline" == snapshot
+
+
+def test_singleline(snapshot):
+    assert "one-line" == snapshot
```

Comment:
Not sure what this assertion adds, we already test single lines above. Is it the `-`?

Reply:
Should also add a test case for python multi lines `"""..."""`